roles/kubernetes/master/templates/kubeadm-config.v1alpha3.yaml.j2

```diff
@@ -7,10 +7,16 @@ nodeRegistration:
 {% if kube_override_hostname|default('') %}
   name: {{ kube_override_hostname }}
 {% endif %}
-{% if inventory_hostname in groups['kube-master'] and inventory_hostname not in groups['kube-node'] %}
+{% if inventory_hostname not in groups['kube-node'] %}
   taints:
   - effect: NoSchedule
+{%   if kube_version is version('v1.14', '<') %}
     key: node-role.kubernetes.io/master
+{%   else %}
+    key: node.kubernetes.io/master
```

Comment:
`node.kubernetes.io/master` has not yet been implemented FYI. So this will not work

Reply:
It should be `node-role.kubernetes.io/master` in all cases currently

Reply:
This looks fixed in the latest commit